pyproject.toml

```diff
@@ -109,6 +109,8 @@ responses = "^0.22.0"
 pytest-asyncio = "^0.20.3"
 lark = "^1.1.5"
 pytest-mock  = "^3.10.0"
+lxml = "^4"
+html2text = "^2020.1.16"
```

Comment:
Required to run the unit tests. `html2text` is currently an optional dependency for langchain and have added lxml as another optional dependency.

Reply:
I'm still in the process of updating the testing workflow so we can handle optional dependencies, so nothing is document yet (sorry about that! :() 

Adding the dependencies here, will make the tests pass, but won't actually work when the langchain package is installed for users (since it's installed without optional dependencies).

Could you move these to the `extended_testing` group below?

```
extended_testing = ["pypdf", "pdfminer.six"]
```

Reply:
Ahhh yes, no trouble at all. That sounds great, this way we don't have duplicated dependency versions in `pyproject.toml` either. I've made this change and added both `lxml` and `html2text` to `extended_testing`.